charts/prometheus-mysql-exporter/README.md

```diff
@@ -41,6 +41,40 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 helm upgrade [RELEASE_NAME] [CHART] --install
 ```
 
+### Multiple-target probes
+
+mysql_exporter now support multi-target probes using the `/probe` route. To enable this feature, set `serviecMonitor.multipleTarget.enabled` to `true` and define your targets in `serviceMonitor.multipleTarget.targets`.
+Credentials for each target should be referenced in the associate config file. Target name should match the entry in the config file.
+As an example, for a config file with two targets:
+
+```yaml
+serviceMonitor:
+  multipleTarget:
+    enabled: true
+    targets:
+      - name: localhost
+        endpoint: 127.0.0.1
+      - name: remote
+        endpoint: 8.8.8.8
+        port: 3307
+```
+
+Config file should have the following entries:
```

Comment:
is it possible to configure this section from values.yaml?

Reply:
It could be possible but should we allow this approach ? I think it's safer to manage secrets externally.

However we can add a new variable `mysql.multiTargetCredentials` and generate our secret if it's not empty.

Reply:
I agree. That's a better way. I was a bit confused between the old configuration and this new configuration.

Reply:
Do you agree regarding that secret should be manage externally or that we should be able to generate our secret base on the values ?

Reply:
> Do you agree regarding that secret should be manage externally

yes I agree. Sorry that my previous comment has confused you.

Reply:
> Do you agree regarding that secret should be manage externally

Yes I agree. My previous comment is confusing, sorry about that.